### ISSUE TITLE
Blank space before headings

### ISSUE THREAD

**Issue by bjb568** [ocf/ocfstatic#170]
<img width="565" alt="Screenshot 2020-09-20 at 19 02 43" src="https://user-images.githubusercontent.com/5396448/93724318-fe087780-fb73-11ea-8841-264a999f310e.png">

The HTML associated with the blank space is `<a href="#installing-wordpress" aria-hidden="true"><span class="icon icon-link"></span></a>`, which makes me think there should be an icon rendering there, but I don't see any CSS that would actually create an icon. Hovering over it makes my pointer become a finger, which I guess is probably intentional.

**Comment by BernardZhao** [comment]
This is also related to #169, bulma is polluting the icon class and causing that space. What is supposed to happen is that link is supposed to be invisible, like if you inspect this example on a demo like one [here](https://gridsome-starter-blog.netlify.app/say-hello-to-gridsome/)
